perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/build.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -e
+sudo apt-get update
+
+echo "Installing git"
+sudo apt-get install git
+echo "Installing pip"
+sudo apt-get install pip -y
+echo "Installing go-lang 1.20.4"
+wget -O go_tar.tar.gz https://go.dev/dl/go1.20.4.linux-amd64.tar.gz
+sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local
+export PATH=$PATH:/usr/local/go/bin
+echo "Installing fio"
+sudo apt-get install fio -y
+echo "Installing docker "
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
+sudo apt-get install jq
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin -y
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse"
+
+# Get the current date and time
+current_date=$(date +"%Y-%m-%d %H:%M:%S")
+
+# Get the required enabled configuration
+config=$(jq --arg EXPERIMENT_NUMBER "$EXPERIMENT_NUMBER" --arg current_date "$current_date" '.experiment_configuration[] | select(.end_date >= $current_date)' perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json | jq -s ".[$EXPERIMENT_NUMBER-1]")
+
+# Check if any enabled configurations were found
+if [ "$config" = "null" ]; then
+  echo "No enabled configuration found for value: $EXPERIMENT_NUMBER"
+  exit 0
+fi
+
+# Access specific properties of the configuration
+CONFIG_NAME=$(echo "$config" | jq -r '.config_name')
+GCSFUSE_FLAGS=$(echo "$config" | jq -r '.gcsfuse_flags')
+BRANCH=$(echo "$config" | jq -r '.branch')
+END_DATE=$(echo "$config" | jq -r '.end_date')
+
+# Checkout to BRANCH to use test scripts in BRANCH.
+git checkout $BRANCH
```

Comment:
I should have mentioned earlier that we don't need to checkout to $BRANCH here to build gcsfuse. As the comment also says "Checkout to BRANCH to use test scripts in BRANCH", this checkout will  be helpful to use test scripts of that branch. In our case, we always want to use the tests scripts of the branch that was configured in Kokoro. So you can remove this line.

Reply:
Right, thanks!